bundles/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/internal/console/PowermaxCommandExtension.java

```diff
@@ -95,7 +101,8 @@ public void execute(String[] args, Console console) {
     @Override
     public List<String> getUsages() {
         return Arrays.asList(new String[] { buildCommandUsage("<bridgeUID> " + INFO_SETUP, "information on setup"),
-                buildCommandUsage("<bridgeUID> " + DOWNLOAD_SETUP, "download setup") });
+                buildCommandUsage("<bridgeUID> " + DOWNLOAD_SETUP, "download setup"),
+                buildCommandUsage("<bridgeUID> " + BRIDGE_STATE, "show current state") });
```

Comment:
I am not sure there is really the need for a new command. Maybe this could be included the info_setup command ?

Reply:
Could go either way. I saw it as: info_setup shows your panel configuration which is basically static information. The new bridge_state command shows the current state which is constantly changing and this is a point-in-time snapshot. So I liked having them in separate commands.

Reply:
Ok